src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -586,6 +586,12 @@ private function getTemplateVarVouchers(Cart $cart)
                 $totalCartVoucherReduction = $this->includeTaxes() ? $cartVoucher['value_real'] : $cartVoucher['value_tax_exc'];
             }
 
+            // Avoid display of a voucher if its total is equal to zero tax included
+            if (0 == $totalCartVoucherReduction) {
```

Comment:
should only use strict comparison `===`. (I think even phpstan wouldn't allow having `==`, but I guess this file is ignored)

Reply:
Thanks @zuk3975 
Let me please just investigate is ever `$totalCartVoucherReduction)` is the string "0.00".
Otherwise, your feedback seems ok to me.

Reply:
if you want to make it perfect, you can use DecimalNumber smth like:
`(new DecimalNumber((string) $totalCartVoucherReduction))->isEqualToZero()`